src/magellan_motion/src/path_planner/planner.cpp

```diff
@@ -33,7 +31,12 @@ PathPlanner::PathPlanner(ros::NodeHandle& nh, double resolution)
           open_(comp_),
           map_sub(nh.subscribe("/grid", 10, &PathPlanner::mapCallback, this))
 {
-    mapSize = 10 / resolution;
+    nh.getParam("~float_precision", floatPrecision);
```

Comment:
You should check that these params are loaded in properly. I.e.
https://github.com/Pitt-RAS/racecar/blob/master/src/magellan_motion/src/path_follower/path_follower_node.cpp#L15